docs/02-app/02-api-reference/02-file-conventions/route-segment-config.mdx

```diff
@@ -197,6 +200,21 @@ Support for `preferredRegion`, and regions supported, is dependent on your deplo
 > - If a `preferredRegion` is not specified, it will inherit the option of the nearest parent layout.
 > - The root layout defaults to `all` regions.
 
+### `maxDuration`
+
```

Comment:
I think it would be nice to have a sentence that explains what this does. See `dynamicParams` and `revalidate`

Reply:
Updated. Not 100% sure about the wording here, happy if you could make suggestions 🙏🏻